Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl

```diff
@@ -0,0 +1,1264 @@
+    dcterms:isFormatOf <http://www.ucc.ie/celt/published/G105003/text003.html>;
+    dcterms:format "text/turtle".
+
+<#LabraidLoingsechMóen>
+    a foaf:Person;
+    irishRel:nomName "Labraid Loingsech Móen";
+    rel:childOf <#AilellaÁine>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#LabraidLoingsech>;
+    rdfs:comment "a quo Lagin ut Laidcenn m. Barcedo dixit".
+
+<#AilellaÁine>
+    a foaf:Person;
+    irishRel:genName "Ailella Áine";
+    irishRel:nomName "Ailill Áine";
+    rel:childOf <#LóegaireLuirc>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#AilillAine>.
+
+<#LóegaireLuirc>
+    a foaf:Person;
+    irishRel:nomName "Lóegaire Lorc";
+    irishRel:genName "Lóegaire Luirc";
+    rel:childOf <#AugaineMáir>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#LóegaireLorcc>.
+
+<#AugaineMáir>
+    a foaf:Person;
+    irishRel:nomName "Augaine Már";
+    irishRel:genName "Augaine Máir";
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#AugaineMár>.
+
+<#LabraidLongsechMóen-d45dbad0>
+    a foaf:Person;
+    irishRel:nomName "Labraid Longsech Móen";
+    rel:childOf <#AilillÁine>;
+    owl:sameAs <#LabraidLoingsechMóen>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Énna_Labraid_luad_cáich>.
+
+<#AilillÁine>
+    a foaf:Person;
+    irishRel:nomName "Ailill Áine";
+    owl:sameAs <#AilellaÁine>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Énna_Labraid_luad_cáich>.
+
+<http://www.vanhamel.nl/codecs/Énna_Labraid_luad_cáich>
+    prov:wasAttributedTo <#Laidcenn>.
+
+<#Laidcenn>
+    a foaf:Person;
+    irishRel:nomName "Laidcenn";
+    rel:childOf <#Barcedo>;
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#Laidcenn>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Énna_Labraid_luad_cáich>.
+
+<#Barcedo>
+    a foaf:Person;
+    irishRel:genName "Barcedo";
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#Bairceda>
+
+<#Labraid>
+    a foaf:Person;
+    irishRel:nomName "Labraid";
+    rel:antagonistOf <#CobthachCóel>;
+    owl:sameAs <#LabraidLoingsechMóen>;
+    rdfs:comment "Is é in Labraid-sin trá ro h-alt h-i tírib Gall & do-dechaid dar muir co n-Gallaib imme dochumm n-Alban co h-Érind.".
+
+<#CobthachCóel>
+    a foaf:Person;
+    irishRel:nomName "Cobthach Cóel";
+    rel:antagonistOf <#LóegaireLorcc>;
+    owl:sameAs <http://example.com/Rawl_B502/genelach_ríg_n_alban.ttl#Cobthaich>;
+    rdfs:comment "Is Cobthach Cóel ro h-ort Lóegaire Lorcc h-i tosuch conid ó h-shain le is chocad eter a clanna .i. Lagin & Leth Cuind ut Orthanach loquitur.".
+
+<#LóegaireLorcc>
+    a foaf:Person;
+    irishRel:nomName "Lóegaire Lorcc";
+    owl:sameAs <#LóegaireLuirc>.
+
+<#Orthanach>
+    a foaf:Person;
+    irishRel:nomName "Orthanach";
+    rdfs:seeAlso <http://vanhamel.nl/codecs/Orthanach_%C3%BAa_C%C3%B3ill%C3%A1ma>;
+    rdfs:comment "[author of 'Cóic cét bliadnae buadach rím']".
```

Comment:
Again, 'Cóic cét bliadnae buadach rím' is unfortunately not sufficiently interesting to have a URL of its own anywhere. If we can create URIs for texts ourselves, then prov:wasAttributedTo could indeed be added.